assets/js/country_support.js

```diff
@@ -108,14 +104,4 @@ function loadCountrySupportTable(elem, fetch) {
     });
 }
 
-/** @type {Promise<Fetch>} */
-const fetchPromise =
```

Comment:
Should we have it show the error screen if it can't find fetch? Just so if 0.01% of people do visit in an old browser they see something?

Reply:
Hm, that's a slippery slope 😅  The `catch` would be enough to handle the absence of Fetch, but if we're talking about IE<=11, it's unlikely that browser would have Promise support.

I'm wondering if there might be some way to initialize the page with some fallback content that'd be replaced in supported browsers. We'd want to be careful to avoid any flashes of content where that fallback content would be briefly visible for the supported case. Maybe combine it with no-JavaScript fallback, e.g. "Your browser is unsupported or JavaScript is disabled."

Reply:
hmm yeah and it looks like our error messaging is "try again later" so that would be a confusing/inaccurate error message. Probably better to just remove this for now and if we get complaints later or see a higher-than-expected % of IE in our Google Analytics, we could revisit